wrench/src/yaml_frame_reader.rs

```diff
@@ -1216,7 +1216,9 @@ impl YamlFrameReader {
         wrench: &mut Wrench,
         yaml: &Yaml,
     ) {
-        let full_clip = LayoutRect::new(LayoutPoint::zero(), wrench.window_size_f32());
+        // The window plus some fudging for nesting in stacking contexts
+        // FIXME: compute this properly?
+        let full_clip = LayoutRect::new(LayoutPoint::zero(), wrench.window_size_f32()).inflate(100.0, 100.0);
```

Comment:
would be great to resolve this FIXME

Reply:
This is a bit tricky, since aiui we can't just keep a running sum of the parent offsets. If a parent has a transform like a rotation we need to, like, compute rotated bounding boxes, I think? Maybe it's simpler than I think though.